### ISSUE TITLE
Library not loaded, image not found

### ISSUE THREAD

**Issue by ronaldoussoren** [ronaldoussoren/py2app#268]
**[Original report](https://bitbucket.org/ronaldoussoren/py2app/issue/268) by Paul Wiseman (Bitbucket: [Paul](https://bitbucket.org/Paul), GitHub: [Paul](https://github.com/Paul)).**

----------------------------------------

Hey,

I’m running into an issue with a built app when running. I’ve packaged wxPython up with py2app before in the past with no problems

```
  ...
  File &#34;wx/__init__.pyc&#34;, line 17, in &lt;module&gt;
  File &#34;wx/core.pyc&#34;, line 12, in &lt;module&gt;
  File &#34;wx/_core.pyc&#34;, line 14, in &lt;module&gt;
  File &#34;wx/_core.pyc&#34;, line 10, in __load
ImportError: dlopen(/Applications/MyApp.app/Contents/Resources/lib/python2.7/lib-dynload/wx/_core.so, 2): Library not loaded: @loader_path/libwx_osx_cocoau_core-3.0.0.4.0.dylib
  Referenced from: /Applications/MyApp.app/Contents/Resources/lib/python2.7/lib-dynload/wx/_core.so
  Reason: image not found
```

I’m running altgraph-0.16.1 macholib-1.11 modulegraph-0.17 py2app-0.19 wxpython \(4.0.6 osx-cocoa \(phoenix\) wxWidgets 3.0.5\) with framework python.org 2.7.16 32/64 bit build

The framework it can’t find is bundled in the app, it just can’t find it at runtime it seems

```
$ ls /Applications/MyApp.app/Contents/Frameworks/
Python.framework            librsync.2.0.2.dylib            libssl.1.0.0.dylib          libwx_baseu_net-3.0.0.4.0.dylib     libwx_osx_cocoau_core-3.0.0.4.0.dylib
libcrypto.1.0.0.dylib           librsync.2.dylib            libwx_baseu-3.0.0.4.0.dylib     libwx_osx_cocoau_adv-3.0.0.4.0.dylib    libwx_osx_cocoau_stc-3.0.0.4.0.dylib
```

**Comment by hasii2011** [comment]
Thanks for moving this over

**Comment by ronaldoussoren** [comment]
This should be fixed in 2fde755a903bb9f20c3095b41f1ef4088f435690.
